explainaboard/analysis/__init__.py

```diff
@@ -1 +1,3 @@
+# flake8: noqa: FI18
```

Comment:
I think importing is better even in `__init__.py`.

Reply:
I'll add the import statement to `__init__.py` as I don't have a strong opinion.

Reply:
Added.